src/app/components/expired-login-link/expired-login-link.component.html

```diff
@@ -0,0 +1,13 @@
+<exui-app-header></exui-app-header>
+<div class="govuk-width-container">
+<main class="govuk-main-wrapper govuk-main-wrapper--l" id="content" role="main">
+    <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h2 class="govuk-heading-xl">{{'We could not log you in' | rpxTranslate}}</h2>
```

Comment:
The user is logged in at this point as far as IDAM is concerned, so I'm worried this wording will lead to some confused support tickets. Could it be changed to "Access Error" or "Your session has expired"?